skill_scanner/core/scanner.py

```
@@ -243,6 +243,48 @@ def _scan_single_skill(self, skill: Skill, skill_directory: Path) -> ScanResult:
+                                "demoted_to": r.demoted_to,
+                                "model_id": r.model_id,
+                            }
+                            for r in adj.audit
+                        ]
+                    else:
+                        logger.debug(
+                            "adjudicator enabled but no LLM model configured; "
+                            "set SKILL_SCANNER_LLM_MODEL or "
+                            "SKILL_SCANNER_ADJUDICATOR_LLM_MODEL to activate"
+                        )
+                except Exception as exc:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)